.github/dependabot.yml

```diff
@@ -1,10 +1,21 @@
 version: 2
 updates:
   - package-ecosystem: "npm"
-    directory: "/"
+    directory: "."
     schedule:
-      interval: "daily"
+      interval: "weekly"
+      day: "saturday"
+    versioning-strategy: increase
+    groups:
+      npm:
+        patterns:
+          - "*"
   - package-ecosystem: "github-actions"
     directory: ".github/workflows"
     schedule:
-      interval: "daily"
+      interval: "weekly"
+      day: "saturday"
+    groups:
+      gha:
+        patterns:
+          - "*"

```

Comment:
Can you explain the changes to this file?

Reply:
I used Dependabot [groups](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups) feature to batch npm and gha dependency updates into two separate PRs which opens every Saturday if there is at least one update.

Reply:
Reverted this change since I realised it was out of scope for this PR. Might add this in subsequent PR though.